apitest/src/main/resources/partner/policy/DefineDataSharePolicy/DefineDataSharePolicy.yml

```diff
@@ -61,26 +63,27 @@ defineDataSharePolicy:
     
    Partner_DefineDataSharePolicy_All_for_Update_sid:
       endPoint: /v1/policymanager/policies
+      description: Create Data Share Policy for update partner
       role: partner
       restMethod: post
       inputTemplate: partner/policy/DefineDataSharePolicy/defineDataSharePolicy
       outputTemplate: partner/policy/DefineDataSharePolicy/defineDataSharePolicyResult
       input: '{
-		"desc": "desc mosip policy group for req key For Update",
```

Comment:
Undo the changes that happened other that description addition

Reply:
Updated changes